dracut/30ignition/module-setup.sh

```diff
@@ -42,18 +51,9 @@ install() {
     inst_simple "$moddir/ignition-generator" \
         "$systemdutildir/system-generators/ignition-generator"
 
-    inst_simple "$moddir/ignition-disks.service" \
-        "$systemdsystemunitdir/ignition-disks.service"
-
     inst_simple "$moddir/ignition-fetch.service" \
         "$systemdsystemunitdir/ignition-fetch.service"
 
-    inst_simple "$moddir/ignition-files.service" \
-        "$systemdsystemunitdir/ignition-files.service"
-
-    inst_simple "$moddir/ignition-mount.service" \
-        "$systemdsystemunitdir/ignition-mount.service"
-
     inst_simple "$moddir/ignition-ask-var-mount.service" \
         "$systemdsystemunitdir/ignition-ask-var-mount.service"
```

Comment:
Should this be moved into the new target?

Reply:
To be clear I only tested this to the point of getting RHCOS rebased on the latest fedora-coreos-config, I didn't yet test it with a live setup. 

Looking at this, probably yes.